pkg/auth/configmap_vault_config_saver.go

```diff
@@ -0,0 +1,96 @@
+package auth
+
+import (
+	"fmt"
+
+	"github.com/jenkins-x/jx/pkg/secreturl"
+	"github.com/pkg/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1 "k8s.io/client-go/kubernetes/typed/core/v1"
+	"sigs.k8s.io/yaml"
+)
+
+const (
+	labelAuthConfig      = "jenkins.io/config-type"
+	labelAuthConfigValue = "auth"
+)
+
+// LoadConfig loads the auth config from a ConfigMap witch stores in its
```

Comment:
`s/witch/which` unless we've got an infestation of magic-using `ConfigMap`s flying around on broomsticks. =)

Reply:
done